src/main/java/com/tstepnik/planner/controller/UserController.java

```diff
@@ -43,12 +43,12 @@ public UserController(UserService userService, AuthService authService, UserMapp
 
     @PutMapping("/{id}")
     @PreAuthorize("hasRole('USER')")
-    public ResponseEntity<UserDTO> updatedUser(@RequestBody UserDTO user, @PathVariable("id") long id) {
-        Optional<User> updateUser = Optional.ofNullable(userService.updateUser(mapper.toUser(user), id));
-        if (updateUser.isEmpty()) {
-            return new ResponseEntity<UserDTO>(mapper.toDto(updateUser.get()), HttpStatus.CONFLICT);//TODO remove when global exception handler is added.
+    public ResponseEntity<UserDTO> updateUser(@RequestBody UserDTO user, @PathVariable("id") long id) {
```

Comment:
Duplicated changes. Seen this in different PR

Reply:
What do you mean ?

Reply:
Exactly what I said, I have the same change in other PR. My question is, do you know why I have seen it in both of them?